### ISSUE TITLE
Accessing the releaseUrl in hooks

### ISSUE THREAD

**Issue by birtles** [release-it/release-it#1038]
Hi!

I am using `release-it` to generate a draft release on GitHub so that I can go in and review/tweak the release notes before publishing it.

`release-it` helpfully prints out the release URL, here I guess: https://github.com/release-it/release-it/blob/e2e0c34cd392f7c87e41300b10c2d398582d32ce/lib/plugin/GitRelease.js#L53

However, after the GitHub action is run, GitHub folds up the action log and it's quite hard to dig through all the logs to find that line.

I wanted to make that URL easier to find so I tried adding a hook to add it to the action summary along the following lines:

```json
  "hooks": {
    "after:release": "echo 'Draft release: ${releaseUrl}' >> $GITHUB_STEP_SUMMARY"
  },
```

However, it looks like `releaseUrl` is not available as I get:

```
ERROR Unable to render template with context:
echo 'Draft release: ${releaseUrl}' >> $GITHUB_STEP_SUMMARY
...
ERROR ReferenceError: releaseUrl is not defined
```

Would it be possible to make this available, or is there another means of doing this?

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v16.2.0. See [Release 16.2.0](https://github.com/release-it/release-it/releases/tag/16.2.0) for release notes._

**Comment by birtles** [comment]
Thank you so much!
